content/SCALE/SCALETutorials/Storage/ManagePoolsScale.md

```diff
@@ -54,32 +56,149 @@ Select **Confirm Export/Disconnect**
+If a pool is not automatically expanded, for example when resizing virtual disks in a hypervisor apart from TrueNAS, click **Expand** on the **Storage Dashboard** to manually increase the pool size to match all available disk space.
+
+### Extending a RAIDZ VDEV
+
+Additional disks can be added one at a time to a RAIDZ VDEV, expanding its capacity incrementally.
+This can be useful for small pools (typically with only one RAID-Z VDEV), where there isn't sufficient hardware capacity to add a second VDEV, doubling the number of disks.
+
+{{< expand "Overview and Considerations" "v" >}}
+TrueNAS SCALE 24.10 (Electric Eel) introduces RAIDZ extension to allow incremental expansion of an existing RAIDZ VDEV using one more disks.
+RAIDZ extension allows resource- or hardware-limited homelab and small enterprise users to expand storage capacity with lower upfront costs compared to traditional ZFS expansion methods.
+
+To expand a RAIDZ array, TrueNAS reads data from the current disks and rewrites it onto the new configuration, including any additional disks.
+
+Data redundancy is maintained.
+Make sure the pool is healthy before beginning the expansion process.
+If a disk fails mid-expansion, the process pauses until the RAIDZ virtual device (vdev) is healthy again, typically by replacing the failed disk and waiting for the system to rebuild.
+
+The storage pool remains accessible throughout the expansion.
+If you reboot or export/import the pool, the expansion resumes from where it left off.
+
+After the expansion, the extra space becomes available for use.
+
+The fault tolerance level of the RAIDZ array remains unchanged.
+For example, a RAIDZ2 stays a RAIDZ2 even after adding more disks.
+
+You can expand a RAIDZ vdev multiple times.
+
+Existing data blocks retain their original data-to-parity ratio and block width, but spread across the larger set of disks.
+New data blocks adopt the new data-to-parity ratio and width.
+Because of this overhead, an extended RAIDZ VDEV can report a lower total capacity than a freshly created VDEV with the same number of disks.
+
+{{< trueimage src="/images/Reference/RaidzExpansion.png" alt="RAIDZ Expansion" id="RAIDZ Expansion" caption="Before (left) and after (right) expansion of a four-disk to five-disk RAIDZ1<br>Thanks to Matt Ahrens ([Source](https://arstechnica.com/gadgets/2021/06/raidz-expansion-code-lands-in-openzfs-master/))" >}}
+
+The additional overhead of now-undersized existing blocks can be recovered by reading and rewriting existing data to the extended pool.
+This can occur naturally over the lifetime of the pool as ZFS rewrites data blocks when you modify or delete data.
+Alternately, you can manually replicate and rewrite the data to the pool.
+Data is rewritten using the extended block size and parity ratio.
```

Comment:
```suggestion
Extended VDEVs recover lost headroom as existing data is read and rewritten to the new parity ratio.
This can occur naturally over the lifetime of the pool as you modify or delete data.
To manually recover capacity, simply replicate and rewrite the data to the extended pool.
```